c2cgeoportal/tests/functional/__init__.py

```diff
@@ -63,65 +64,65 @@
 caching.init_region({"backend": "dogpile.cache.memory"})
 
 
+def cleanup_db():
```

Comment:
That clean really all ?  What about metadata, tsearch... ? Perhaps layer ? (or that comes with treeitem ? but both layerv1 and v2 ?)